README.md

```diff
@@ -9,7 +9,7 @@ project.
 [![Crates.io](https://img.shields.io/crates/v/ratatui?logo=rust&style=flat-square)](https://crates.io/crates/ratatui)
 [![License](https://img.shields.io/crates/l/ratatui?style=flat-square)](./LICENSE) [![GitHub CI
 Status](https://img.shields.io/github/actions/workflow/status/ratatui-org/ratatui/ci.yml?style=flat-square&logo=github)](https://github.com/ratatui-org/ratatui/actions?query=workflow%3ACI+)
-[![Docs.rs](https://img.shields.io/docsrs/ratatui?logo=rust&style=flat-square)](https://docs.rs/crate/ratatui/)  
+[![Docs.rs](https://img.shields.io/docsrs/ratatui?logo=rust&style=flat-square)](https://docs.rs/crate/ratatui/)
```

Comment:
Unrelated formatting change? 😅

Reply:
Resolved in https://github.com/ratatui-org/ratatui/pull/446/commits/905fe7a21cb2c9eebae20e210d76df941f53f8f8

I did turn off my markdown formatter for this PR but I have a git pre-commit hook that trims all whitespaces, and didn't catch that while `git add` + `git commit`ing :)